### ISSUE TITLE
Control or Destroy

### ISSUE THREAD

**Issue by B1naryGaming** [siggame/MegaMinerAI-Game-Ideas#53]
# **The game:**
## **TLDR**
- Victory points are gained by controlling territory with buildings
- You can also win by capturing the enemies HQ
- 40x40 map. (different types of terrain)
- 4 types of units: Soldiers, commanders, snipers, artillery (motor).
- Gold is gained based on the number of buildings.
- you can build buildings
- units that are stationary gain extra bonus defense.
- air support (auto destroy a building and units except for HQ)

## **Victory:**
The game continues until you destroy the enemy HQ or X turns have passed.
If X turns have passed, the player with the most territory wins (buildings count as 3).
Otherwise, it is whoever has the HQ with the highest HP.
Otherwise, it is whoever has the most value.
Otherwise, it is a coin flip.

## **Map:**
A random 40x40 map with various terrains like mountains and swamps that would impede unit speed and building. Raised terrain to possibly give an advantage to ranged units.

## **Buildings:**
Every player starts with an HQ where all units are made.
HQ grants 1 gold per 2 turns.
possible buildings:

| Name | HP | gold gen | Defence | cost |
|----|----|----|----|----|
|**Tower**|50|5|10|50GP|
|**Bunker**|200|10|20|250GP|
|**Berracks**|100|20|15|500GP|
|**sandbags**|50|0|15|50GP|
|**Residential**|100|5|15|N/A|

## **Units:**
10 units per tile. (more in buildings?)
Units auto attack all enemies in range. They must be ordered to attack and destroy buildings.
All units spawn at HQ or barracks
Units and buildings gain a range bonus.
(bonus gained in tower or residential)

| Name | HP | Damage | movement | Range | cost |
|----|----|----|----|----|----|
|**Soldier**|5|1|3|2|5GP|
|**Commander**|10|2|3|3|10GP|
|**Sniper**|3|3|2|5|20GP|
|**Artilary**|5|5|1|4|25GP|
|**Tank**|20|10|2|10|50GP|

## **Combat + infamy:**
Each unit spreads it's damage out across each tile within range, and any defense is subtracted from it. From there it is spread throughout the units.
Players may spend 500 gold every X turns to destroy a building and all units in it.
